src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -289,6 +289,20 @@ const MetaTrader = (() => {
                 }
 
                 const req = makeRequestObject(acc_type, action);
+                if (action === 'new_account' && MetaTraderUI.shouldSetTradingPassword()) {
+                    const resp = await BinarySocket.send({ mt5_login_list: 1 });
```

Comment:
(suggestion/question) Possible to handle the error case here? also using spread to define a better naming for response? 
```suggestion
                    const {mt5_login_list, error} = await BinarySocket.send({ mt5_login_list: 1 });
                    if (error) // needs handling? 
```

Reply:
@reza-fs spread handling is applied but I can't think of an error handling in this phase. because this is just getting MT5 login list without any user-generated or dynamic payload and so it is almost error-proof!
By the way, we are not handling errors in many other places `mt5_login_list` is called and I think it is OK